.github/workflows/ci.yaml

```diff
@@ -0,0 +1,71 @@
+name: CI
+
+on: [workflow_dispatch, pull_request]
+
+env:
+  SECRET_KEY: thisisasecret
+  DATABASE_URL: postgresql://postgres:password@localhost:5432/postgres
+
+jobs:
+  ci:
+    strategy:
+      matrix:
+        python-version: ["3.6"]
+        platform: [ubuntu-20.04]
+
+    runs-on: ${{ matrix.platform }}
+
+    services:
+      postgres:
+        image: postgres:11-alpine
+        ports:
+          - 5432:5432
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: password
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v4
+      with:
+        python-version: ${{ matrix.python-version }}
+        cache: 'pip'
+
+    - name: Set up Python dependencies
+      run: |
+        pip install -U pip
+        # pip install wheel
+        # pip wheel -q -r requirements/travis.txt
+        pip install -r requirements/travis.txt
+
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 12
+
+    - name: Setup Yarn install
+      uses: borales/actions-yarn@v4
+      with:
+        cmd: install
+
+    - name: Run Flake8 on src
+      run: |
+        flake8 src
+
+    - name: Run DB migration for test
+      run: |
+        python src/manage.py migrate --verbosity 1 --noinput
+
+    - name: Run Test and Codecov
```

Comment:
I'd suggest adopting the [official codecov action](https://github.com/codecov/codecov-action).
You can add a `TODO` comment here if you don't want to handle it with this PR.

Reply:
added in 12fef348472607ecc7e7003f086adc5af7ca8d85
ran it a couple of times finding it couldn't locate the coverage file, on the last change, I straight assign the coverage file to the action. It seems to work. But how can I verify it? Is it supposed to have a new report on the conversation?